### ISSUE TITLE
Wrong type for `interaction.data.options`

### ISSUE THREAD

**Issue by Kayyow** [Kraigie/nostrum#434]
I think the `options` key of the `ApplicationCommandInteractionData` struct is wrongly typed and it causes me Dialyzer errors. It should be a list but it is instead a single `ApplicationCommandInteractionDataOption` struct.

How `options` is currently typed :
```ex
@type options :: ApplicationCommandInteractionDataOption.t() | nil
```

How I think it should be typed :
```ex
@type options :: [ApplicationCommandInteractionDataOption.t()] | nil
```

I'm still new with Elixir so maybe there is something that I don't understand correctly.

**Comment by cedeno** [comment]
made PR https://github.com/Kraigie/nostrum/pull/436

**Comment by jchristgit** [comment]
Fixed by #436, thank you for the report
